meshroom/core/desc.py

```diff
@@ -589,6 +593,23 @@ def processChunk(self, chunk):
         finally:
             chunk.subprocess = None
 
+#specific command line node for alicevision apps
+class AVCommandLineNode(CommandLineNode):
+    def buildCommandLine(self, chunk):
+
+        str = super(AVCommandLineNode, self).buildCommandLine(chunk)
+
+        cmdMem = ''
+        memSize = cgroup.getCgroupMemorySize()
```

Comment:
This is a static information that should be estimated once for the lifetime of the session and not each time we evaluate the command line per node.